elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -90,26 +90,19 @@ export default class BulbsDfp extends BulbsHTMLElement {
 
   handleInterval () {
     util.getAnalyticsManager().sendEvent({
-      eventCategory: 'bulbs-dfp-element Metrics',
+      eventCategory: 'bulbs-dfp-element Hot Metrics',
```

Comment:
If it makes more sense, this could have the same `eventCategory`, and change the `eventAction`:
```
eventAction: '30-second-refresh-triggered',
```
becomes
```
eventAction: 'live:30-second-refresh-triggered',
```